src/cryptsetup/cryptsetup.c

```diff
@@ -2,6 +2,7 @@
   This file is part of systemd.
 
   Copyright 2010 Lennart Poettering
+  Copyright 2016 Johanna Abrahamsson (PKCS#11 part)
```

Comment:
No need to add the part in parentheses. We have git blame to determine who wrote what, and comments like this tend to get out of date very quickly.